### PR TITLE
Morpho eUSD Plugin (Base)

### PR DESCRIPTION
**Pull request by julianmrodri**
Metamorhp Morpho EUSD Pluing on Base

### CODE REVIEW

File: scripts/deployment/phase2-assets/collaterals/deploy_morphoeUSD.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    ).deploy(
+      {
+        priceTimeout: PRICE_TIMEOUT.toString(),
+        chainlinkFeed: eUSD_USD_FEED,
+        oracleError: eUSD_ORACLE_ERROR.toString(),
+        erc20: networkConfig[chainId].tokens.meUSD,
+        maxTradeVolume: fp('1e6').toString(), // 17m vault
+        oracleTimeout: eUSD_ORACLE_TIMEOUT.toString(),
+        targetName: hre.ethers.utils.formatBytes32String('USD'),
+        defaultThreshold: eUSD_ORACLE_ERROR.add(fp('0.01')).toString(), // +1% buffer rule
```
Comment (pmckelvy1) [inline scripts/deployment/phase2-assets/collaterals/deploy_morphoeUSD.ts:68]: do we need to add a full 1% here? we're targetting base, so the oracles should be quick to update, and the base oracle error is 0.5%. feels like we could get away with 1% total? (oracle error + 0.5%). @tbrent thoughts?

File: scripts/deployment/phase2-assets/collaterals/deploy_morphoeUSD.ts [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        delayUntilDefault: DELAY_UNTIL_DEFAULT.toString(),
+      },
+      fp('1e-4') // can have mild drawdowns
```
Comment (pmckelvy1) [inline scripts/deployment/phase2-assets/collaterals/deploy_morphoeUSD.ts:71]: morpho drawdowns are this big?!

Comment (tbrent) [reply scripts/deployment/phase2-assets/collaterals/deploy_morphoeUSD.ts:71]: We actually used 10x this on mainnet for Re7WETH after backtesting and finding it defaulted below that:

https://github.com/reserve-protocol/protocol/blob/14da963751b59faff6fb6081fcbdcfe8b99e9b2a/scripts/deployment/phase2-assets/collaterals/deploy_re7weth.ts#L68

File: scripts/deployment/phase2-assets/collaterals/deploy_morphoeUSD.ts (hunk 1 above)
Comment (tbrent) [reply scripts/deployment/phase2-assets/collaterals/deploy_morphoeUSD.ts:68]: The +1% rule doesn't interact with the attributes of the oracle in my mind, eg oracleError or oracleTimeout, etc. 

Anyway, the oracleTimeout is 24h. I don't see why it being on base would change anything, this is still the value that governs how often they will push heartbeat updates.

File: scripts/deployment/phase2-assets/collaterals/deploy_morphoeUSD.ts (hunk 2 above)
Comment (tbrent) [reply scripts/deployment/phase2-assets/collaterals/deploy_morphoeUSD.ts:71]: 🤞 hoping `1e-4` is enough for eUSD

Comment (pmckelvy1) [reply scripts/deployment/phase2-assets/collaterals/deploy_morphoeUSD.ts:71]: i think we have backtesting on base? ill see if we can run this

**Review by pmckelvy1 — APPROVED** [review]
lgtm

**Pull request merged**